### ISSUE TITLE
New trace events

### ISSUE THREAD

**Issue by Sahnvour** [aras-p/ClangBuildAnalyzer#76]
It looks like clang 13 added new events to trace its backend more closely, and ClangBuildAnalyzer does not recognise them.
For example:

```console
WARN: unknown trace event 'AlwaysInlinerPass' in 'redacted.cpp.json', skipping.
WARN: unknown trace event 'ModuleToFunctionPassAdaptor' in 'redacted.cpp.json', skipping.
WARN: unknown trace event 'ModuleToFunctionPassAdaptor' in 'redacted.cpp.json', skipping.
WARN: unknown trace event 'AddressSanitizerPass' in 'redacted.cpp.json', skipping.
WARN: unknown trace event 'AddressSanitizerPass' in 'redacted.cpp.json', skipping.
...
```

Should they just be added as fallthrough after https://github.com/aras-p/ClangBuildAnalyzer/blob/4ab14f2dd0c51f306515285ebd877da2429e360e/src/BuildEvents.cpp#L337 ?

**Comment by sigiesec** [comment]
At least some of them are not that new. See #56

**Comment by aras-p** [comment]
Should be fixed by #80 (merged into main branch, I'll tag the 1.2.0 release binaries soon)
